### ISSUE TITLE
Scan catch-block for Antipattern "log and re-thrown"

### ISSUE THREAD

**Issue by OllisGit** [TNG/ArchUnit#1120]
Hi, 
I want to scan my java code against this anti-pattern "log and re-throw"  (also scanned by sonar: https://rules.sonarsource.com/java/RSPEC-2139).

![image](https://github.com/TNG/ArchUnit/assets/24205767/daa9a39b-fddc-4d50-97d8-72a11c3e3289)

I saw that you deliver a TryCatchBlock-class, but unfortunately it delivers only informations about the try-block and the exception that should be catched.

```
javaClass.getMethods()..getTryCatchBlocks()
```

The sourcecode-informations (parsed code or line-numbers) of the catch-block is not provided.

Any idea how to analyse such code-block?

**Comment by codecholeric** [comment]
I don't think this is possible right now 🤔 As you said, for this we'd have to extend the try-catch-block scanning to also include what's in the catch block, but right now that's not implemented yet.

**Comment by cavram737** [comment]
Hello! I wanted to ask if there has been any update to this topic to access the catch block?
